include/api.php

```diff
@@ -182,7 +181,7 @@ function api_login(App $a)
 		list($consumer, $token) = $oauth1->verify_request($request);
 		if (!is_null($token)) {
 			$oauth1->loginUser($token->uid);
-			Hook::callAll('logged_in', $a->user);
```

Comment:
Is the hook "logged_in" in use by some addon? If yes, we cannot easily remove it.

Reply:
It is called in `Session::setAuthenticatedForUser` that is now called in `FKOAuth1->loginUser`, making this call redundant.